README.md

```diff
@@ -1,26 +1,48 @@
+<!-- markdownlint-disable -->
 # Overview
 
-The "MODULE_NAME" module...
+The Carbon.Windows.HttpServer module contains functions for configuring SSL/TLS/HTTPS certificate bindings on IP
+addresses and ports. Processes can't use an HTTPS certificate unless that certificate has been bound to a IP address and

```

Comment:
Would it be more accurate to say the processes uses the certificate from the certificate store? (There's nothing that stops a process from reading its certificate and private key from a file if it wanted to.)

> In order for a process to use an HTTPS certificate from the Windows certificate store, it must first be bound to an IP address and port.